src/extension.ts

```diff
@@ -33,6 +33,16 @@ export interface ISchemaAssociation {
   uri: string;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
This is also all adapted from [jsonClient.ts](https://github.com/microsoft/vscode/blob/94c9ea46838a9a619aeafb7e8afd1170c967bb55/extensions/json-language-features/client/src/jsonClient.ts)